core/interfaces/status.py

```diff
@@ -112,6 +123,11 @@ class StatusInterface(ui.Interface, GitCommand):
     {}
     """
 
+    def __init__(self, *args, **kwargs):
+        self.conflicts_keybindings = \
+            "\n".join(line[2:] for line in self.conflicts_keybindings.split("\n"))
```

Comment:
Is this line just not readable. 
I would use https://docs.python.org/3.5/library/textwrap.html#textwrap.dedent when I defining the variable instead.

Reply:
Good point. This line was also copied from the rebase dashboard
https://github.com/divmain/GitSavvy/blob/8011d33e935470df7f5d4a631fa4c7ddd14d06db/core/interfaces/rebase.py#L131-131

Reply:
Are you gonna make a PR for this?

Reply:
I created the new PR, but it doesn't yield the desired result
<img width="437" alt="screen shot 2018-02-17 at 4 28 32 pm" src="https://user-images.githubusercontent.com/1690993/36345786-a9659d06-13ff-11e8-88f8-4c92f486726d.png">


It was because we actually do not want to dedent, we just want to remove two spaces from each line.

Reply:
Hm, is there a reason for why we have the two spaces at the beginning of every line?
I think we should remove those and id we want to add some more space to the left side we can increase margin in  https://github.com/divmain/GitSavvy/blob/3c5b1ab49adc9b223d0a461d3e353a2e5cbfc64b/syntax/status.sublime-settings#L9

Reply:
You are probably right but it will be a huge change since a lot of code was written in a form of `line[n:]` where `n` is the number of preceding spaces.

Reply:
hm, it should only be in this one file core/interfaces/status.py

I would change it and then keep it in dev for 2-3 weeks for us to test.